docs/howtos/DataPlane Generation - DPG/09tsp_client.mdx

```diff
@@ -0,0 +1,182 @@
+# Getting started with `tsp-client`
+
+`tsp-client` is a simple command line tool to facilitate generating client libraries from TypeSpec.
+
+## Installation
+
+```
+npm install -g @azure-tools/typespec-client-generator-cli
+```
+
+## Usage
+
+```
+tsp-client <command> [options]
+```
+
+## Commands
+
+Use one of the supported commands to get started generating clients from a TypeSpec project.
+This tool will default to using your current working directory to generate clients in and will
+use it to look for relevant configuration files. To specify a different output directory, use
+the `-o` or `--output-dir` option.
+
+### init
```

Comment:
nit: it might be more clear if you do `tsp-client init` instead of just `init` in the header. That way you can immediately see what the whole command is

Reply:
I was wondering if that could be confusing? Some of these commands have required flags, so I wouldnt want someone to think that they can run the command without them.